src/mds/chunkserverclient/chunkserver_client.cpp

```diff
@@ -62,20 +65,28 @@ int ChunkServerClient::DeleteChunkSnapshotOrCorrectSn(
     ChunkService_Stub stub(channelPtr.get());
 
     brpc::Controller cntl;
-    cntl.set_timeout_ms(rpcTimeoutMs_);
+    cntl.set_timeout_ms(option_.rpcTimeoutMs);
 
     ChunkRequest request;
     request.set_optype(CHUNK_OP_TYPE::CHUNK_OP_DELETE_SNAP);
     request.set_logicpoolid(logicalPoolId);
     request.set_copysetid(copysetId);
     request.set_chunkid(chunkId);
     request.set_correctedsn(correctedSn);
+    // add auth token
+    auto isGet = option_.authClient->GetToken(CHUNKSERVER_ROLE,
```

Comment:
authClient use rpc to get token from mds, but it is mds itself, mds do not need send rpc to himself.